$NetBSD$

Fix failed build (libavutil/common.h depends on UINT64_C macro, which isn't
supposed to be exposed in C++)

--- akode/plugins/ffmpeg_decoder/ffmpeg_decoder.cpp~	2011-06-19 03:08:09.000000000 +0000
+++ akode/plugins/ffmpeg_decoder/ffmpeg_decoder.cpp
@@ -18,6 +18,10 @@
     Boston, MA 02110-1301, USA.
 */
 
+#ifdef __NetBSD__
+#define __STDC_CONSTANT_MACROS
+#endif
+
 #include "akodelib.h"
 // #ifdef HAVE_FFMPEG
 
